microcosm_postgres/dag.py

```diff
@@ -166,3 +215,18 @@ def update_nodes(self, dag):
 
         """
         return dag
+
+
+class StoreDAGCloner(DAGCloner):
+    """
+    An extension of DAGCloner that uses store binding convention
+    to automatically convert a deserialized nodes back into models.
+
+    """
+
+    def to_model(self, model_alias, node):
```

Comment:
Model translation can now be defined explicitly, or concluded from the stores by using this subclass. I think we agreed that after a bit more trial and error we should add a dedicated cloning library and do smarter things with the store/controller/resource - @jessemyers is this acceptable for now, or should I jus implement in the services for now and generalize later?

Reply:
I think implementing in the services is wiser until we have 2+ examples.